### PR TITLE
Pre release for keeping track of V2 contract changes

### PR DESCRIPTION
**Pull request by souradeep-das**
Can we keep a log of the breaking changes here?

### CODE REVIEW

**Comment by InoMurko** [comment]
OH I THOUGHT YOU ADDED CONSOLE.LOG

File: CONTRACTS_V2_LOG.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# CONTRACTS_V2_LOG
```
Comment (InoMurko) [inline CONTRACTS_V2_LOG.md:1]: does keeping this log make much sense? isn't a diff of master - v2.0.0 the same thing?

Perhaps we should start logging *integration changes* as a consequence of moving to v2, as per @boolafish ?

Comment (souradeep-das) [reply CONTRACTS_V2_LOG.md:1]: Yess, sounds good! Lets remove the PR list part if we don't want it and increase upon the integrations section? This was just template-ish

**Comment by thec00n** [comment]
From a security perspective we need a list of all changes that are made to the contracts and narrative on why they are made. We can follow the same format we currently use in the release notes.

**Review by thec00n — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,67 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Pre-Release V2
+
+Process Exit Bounty - [2020-08-31]
+Incentivize calling process Exits by rewarding the caller with a bounty provided for by the exit owner
```
Comment (kevsul) [inline CHANGELOG.md:11]:
```suggestion
Incentivize calling processExit by rewarding the caller with a bounty provided by the exit owner  
```

**Review by kevsul — APPROVED** [review]
LGTM, one minor change to text

**Pull request merged**